doc/api/errors.md

```diff
@@ -819,6 +837,11 @@ For example: `Buffer.write(string, encoding, offset[, length])`
 <a id="ERR_PARSE_HISTORY_DATA"></a>
 ### ERR_PARSE_HISTORY_DATA
 
+<a id="ERR_RENDER_HEADERS_FAILED"></a>
+### ERR_RENDER_HEADERS_FAILED
+
+Used when http headers has been sent before rendering the headers.
```

Comment:
`Used when new HTTP headers are added to a response, after the headers part has already been sent.`